### PR TITLE
feat: update resources definitions extra fields for services

### LINKED ISSUE #10094: remove create_time and update_time for resources in admin api
# Summary information

1. Documentation: Remove the field description that needs to pass in the creation time and update time when creating.
2. delete update_time and create_time from define of admin API .
3. Certificate information is parsed from the certificate body without additional storage.

Actually. we need confrim all of the core resource.

- [x] consumer
- [x] consumer_group
- [x] discovery_nodes
- [x] global_rule
- [x] plugin_config
- [x] plugins
- [x] proto
- [x] route
- [x] service
- [x] ssl
  validity_end
  validity_start
  exptime
- [x] stream_route
- [x] upstream

### PR DESCRIPTION
**Pull request by yuluo-yx**
### Description

- Fixes #10094
- Delete the create_time and update_time field in services;
- Passed compatibility testing;
- Add test cases.

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: apisix/schema_def.lua
Comment (Sn0rt) [inline apisix/schema_def.lua:1]: pls remove the conf field from the service define

**Review by Sn0rt — CHANGES_REQUESTED** [review]

File: apisix/schema_def.lua
Comment (yuluo-yx) [reply apisix/schema_def.lua:1]: > pls remove the conf field from the service define

done!

File: t/script/script_distribute.t
Comment (Sn0rt) [inline t/script/script_distribute.t:1]: I think we not need this feature anymore. can we consider delete whole this file.

**Review by Sn0rt — CHANGES_REQUESTED** [review]

File: t/script/script_distribute.t
Comment (yuluo-yx) [reply t/script/script_distribute.t:1]: Maybe we can delete the entire `t\script` folder. According to the issue description and code file content, this folder is all about script script tests.

**Pull request closed without merging**